plat/arm/board/fvp/fvp_trusted_boot.c

```diff
@@ -1,13 +1,18 @@
 /*
- * Copyright (c) 2016, ARM Limited and Contributors. All rights reserved.
+ * Copyright (c) 2016-2017, ARM Limited and Contributors. All rights reserved.
  *
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
 #include <assert.h>
+#if USE_TBBR_DEFS
+#include <tbbr_oid.h>
+#else
 #include <platform_oid.h>
+#endif
```

Comment:
You are right that the ARM platform code changes are not essential. They can be done separately by someone else if you prefer not to.